userspace/falco/app/actions/init_inspectors.cpp

```diff
@@ -26,7 +26,18 @@ using namespace falco::app;
 using namespace falco::app::actions;
 
 static void init_syscall_inspector(falco::app::state& s, std::shared_ptr<sinsp> inspector) {
-	inspector->set_buffer_format(s.options.event_buffer_format);
+	sinsp_evt::param_fmt event_buffer_format = sinsp_evt::PF_NORMAL;
+	if(s.options.print_base64) {
+		falco_logger::log(falco_logger::level::WARNING,
+		                  "The -b/--print-base64 option is deprecated and will be removed. Use -o "
+		                  "buffer_format_base64=true instead.");
+		event_buffer_format = sinsp_evt::PF_BASE64;
+	}
```

Comment:
Question for a followup: shouldn't that warning rather be printed when parsing the options ?

That way, you can keep all the deprecation related code in one single place.